etc/skanlite.profile

```diff
@@ -12,18 +12,20 @@ include /etc/firejail/disable-passwdmgr.inc
 include /etc/firejail/disable-programs.inc
 
 caps.drop all
-netfilter
+# net none
```

Comment:
Not sure how much we can gain from it (it looks skanlite crashes on some setups if the abstract dbus socket is absent, so if it doesn't crash `net none` should be the safest option). On the other hand it certainly doesn't harm to keep it in.